openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -162,3 +174,38 @@ def test_save_related_send_emails(request, webinar_admin_instance, webinar, mock
         mock_send_webinar_emails.assert_called_once_with(
             MandrillClient.WEBINAR_CREATED, webinar, []
         )
+
+
+def test_readonly_user_admin_change_permission(readonly_user_admin):
+    """
+    Test if ReadOnlyUserAdmin does not have any change permissions
```

Comment:
"Test that" sound more appropriate instead of "Test if"

Please update all occurences.

Reply:
Done!